api/python/provisioner/config.py

```diff
@@ -83,7 +84,9 @@
 PRVSNR_CORTX_DEPS_ISO = (
     PRVSNR_CORTX_REPOS_BASE_DIR / f'{CORTX_3RD_PARTY_ISO_DIR}.iso'
 )
-
+PRVNSR_PIP_ISO_DIR = (
+    PRVSNR_CORTX_REPOS_BASE_DIR / f'{CORTX_PIP_ISO_DIR}.iso'
```

Comment:
as I understand we are not going to have (to support for now)  a standalone python repo ISO assuming only a directory inside a single ISO (either locally or remotely mounted).

So, I think we don't need that for now.